metadata-ingestion/src/datahub/ingestion/source/elasticsearch.py

```diff
@@ -0,0 +1,148 @@
+import logging
+from dataclasses import dataclass, field
+from typing import Iterable, List, Optional
+
+from elasticsearch import Elasticsearch
```

Comment:
`elasticsearch` is not currently part of our venv. We need to add an elasticsearch plugin and add a dependency on this package via matadata-ingestion/setup.py.